app/controllers/ctc/portal/client_logins_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Ctc::Portal::ClientLoginsController < Portal::ClientLoginsController
+  helper_method :wrapping_layout
+
+  private
+
+  def service_type
+    :ctc
+  end
+
+  def wrapping_layout
```

Comment:
It allows us to share the inner parts of the layout (like yes/no questions, intake layout etc) but "wrap" them with the CTC header and footer. Because we inherit from a bunch of GYR classes, we cant inherit from a CTC base controller to define these, so each "top level" controller needs to define it.